pom.xml

```diff
@@ -216,7 +216,8 @@
     <gwt.compiler.localWorkers>1</gwt.compiler.localWorkers>
     <!-- property for productisation to know the last released version -->
     <latestReleasedVersionFromThisBranch>notYetReleased</latestReleasedVersionFromThisBranch>
-    <version.io.takari.maven.plugins>1.13.5</version.io.takari.maven.plugins>
+    <version.io.takari.maven.plugins>1.11.11.jbossorg-1</version.io.takari.maven.plugins>
```

Comment:
why this `1.11.11.jbossorg-1` version? what if we just don't override it? wasn't supposed to used the already defined one?

Reply:
In some jenkins machine (prod) is replaced with the .11.11.jbossorg-1 version, we could leave the 1.13.5 on both and add the comment about why two uses the same version